apis/managedkafka/v1alpha1/topic_types.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	commonv1alpha1 "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/common/v1alpha1"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/k8s/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var ManagedKafkaTopicGVK = GroupVersion.WithKind("ManagedKafkaTopic")
+
+// ManagedKafkaTopicSpec defines the desired state of ManagedKafkaTopic
+// +kcc:proto=google.cloud.managedkafka.v1.Topic
+type ManagedKafkaTopicSpec struct {
+	commonv1alpha1.CommonSpec `json:",inline"`
+
+	// +required
+	Location string `json:"location"`
```

Comment:
Suggest adding descriptions for Location and ClusterRef, because these comments are the source to generate the reference doc, which is the most straightforward (if not only) way for users to configure the KCC object

Reply:
Done! [c6056a0](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3585/commits/c6056a0cdb1644a46d9ea09463c5cccb57071c34)